### PR TITLE
Update tts_de.py

### PR DESCRIPTION
**Pull request by rcbebo82**
Some Translation Errors corrected.

### CODE REVIEW

File: radio/util/tts_de.py [hunk 1]
```diff
@@ -18,63 +18,63 @@
...
                             (u"g", "g0"),
-                            (u"degree", "degree0"),
+                            (u"Grad", "degree0"),
                             (u"radians", "rad0"),
                             (u"Milliliter", "ml0"),
                             (u"Unze", "founce0"),
                             (u"Uhr", "hour0"),
                             (u"Minuten", "minute0"),
-                            (u"Secunden", "second0"),
+                            (u"Sekunden", "second0"),
```
Comment (3djc) [inline radio/util/tts_de.py]: Aren't those 3 plural ? If yes they should be like:
(u"Minute", "minute0"), (u"Minuten", "minute1"),

Comment (rcbebo82) [inline radio/util/tts_de.py]: Of Course. But where is "Stunde", "Stunden"?

File: radio/util/tts_de.py [hunk 2]
```diff
@@ -18,63 +18,65 @@
...
                             (u"g", "g0"),
-                            (u"degree", "degree0"),
+                            (u"Grad", "degree0"),
                             (u"radians", "rad0"),
                             (u"Milliliter", "ml0"),
                             (u"Unze", "founce0"),
                             (u"Uhr", "hour0"),
-                            (u"Minuten", "minute0"),
-                            (u"Secunden", "second0"),
+                            (u"Minute", "minute0"),
```
Comment (3djc) [inline radio/util/tts_de.py]: Sorry, I'm not been painfull on purpose, but can I ask you to keep one line per 'item', like this:
(u"Minute", "minute0"),  (u"Minuten", "minute1"),
(u"Sekunde", "second0"), (u"Sekunden", "second1"),

It allows to quickly sport those which have plural forms from the others

File: radio/util/tts_de.py (hunk 1 above)
Comment (3djc) [inline radio/util/tts_de.py]: Well, it will be there when you add it (I think previous translater did put "Uhr"

**Pull request merged**